boards/arm/arduino_opta_m4/arduino_opta_m4.dts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2023 Felipe Neves
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/h7/stm32h747Xi_m4.dtsi>
+#include <st/h7/stm32h747xihx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/delete-node/ &flash1;
+
+/ {
+	model = "Arduino OPTA M4 core Programmable Logic Controller";
+	compatible = "Arduino,opta";
+
+	/* For Arduino OPTA we use 512K of flash instead of 1MB in order
+	 * to left more space for the PLC Runtime.
+	 */
+	soc {
+		flash-controller@52002000 {
+			flash1: flash@8180000 {
+				reg = <0x08180000 DT_SIZE_K(512)>;
+			};
+		};
+	};
+
+	chosen {
+		zephyr,sram = &sram1;
+		zephyr,flash = &flash1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		status_led_1:led_1 {
+			gpios = <&gpioi 0 GPIO_ACTIVE_LOW>;
+			label = "Status LD1";
```

Comment:
these are not really used for anything anymore AFAIK, unless you want the labeles set for whatever reasons I suggest dropping them, makes it easy to copypaste the block around without stale data